diff --git i/buildSrc/src/main/kotlin/kotlin.gradle.kts w/buildSrc/src/main/kotlin/kotlin.gradle.kts
index 101c9c7..93b1b1a 100644
--- i/buildSrc/src/main/kotlin/kotlin.gradle.kts
+++ w/buildSrc/src/main/kotlin/kotlin.gradle.kts
@@ -9,8 +9,8 @@ repositories {
     mavenCentral()
 }
 
-kotlin { jvmToolchain(8) }
+kotlin {}
 
 tasks.withType<KotlinCompile>().configureEach {
     compilerOptions { freeCompilerArgs.add("-Xcontext-receivers") }
-}
\ No newline at end of file
+}
diff --git i/intellij/build.gradle.kts w/intellij/build.gradle.kts
index 54858f7..5ce2bca 100644
--- i/intellij/build.gradle.kts
+++ w/intellij/build.gradle.kts
@@ -10,7 +10,7 @@ intellij {
     type.set("IU")
 }
 
-kotlin { jvmToolchain(17) }
+kotlin {}
 
 dependencies {
     implementation(project(":project"))
@@ -26,4 +26,4 @@ tasks {
     buildSearchableOptions {
         enabled = false
     }
-}
\ No newline at end of file
+}
diff --git i/lsp/build.gradle.kts w/lsp/build.gradle.kts
index 987eeae..9e4b4cd 100644
--- i/lsp/build.gradle.kts
+++ w/lsp/build.gradle.kts
@@ -41,4 +41,4 @@ tasks {
 
     clean { delete(copyBuiltLsp) }
     build { dependsOn(copyBuiltLsp) }
-}
\ No newline at end of file
+}
diff --git i/settings.gradle.kts w/settings.gradle.kts
index 2b1da4f..b90943b 100644
--- i/settings.gradle.kts
+++ w/settings.gradle.kts
@@ -1,7 +1,3 @@
 rootProject.name = "coachtaal-tooling"
 
-plugins {
-    id("org.gradle.toolchains.foojay-resolver-convention") version "0.8.0"
-}
-
-include("parser", "app", "lsp", "project", "intellij")
\ No newline at end of file
+include("parser", "app", "lsp", "project")
